PopApp/Components/Witness.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  StyleSheet, View, Button, FlatList,
+} from 'react-native';
+import STRINGS from '../res/strings';
+import PROPS_TYPE from '../res/Props';
+import { Buttons, Typography } from '../Styles';
+import Attendee from './Attendee';
+
+/**
+* The Witness component
+*
+* Manage the Witness screen
+*/
+const styles = StyleSheet.create({
+  container: {
+  },
+  text: {
+    ...Typography.base,
+  },
+  button: {
+    ...Buttons.base,
+  },
+});
+
+const Witness = ({ navigation }) => (
+  <FlatList
```

Comment:
Both `data` and `renderItem` attributes must be specified in a `FlatList` according to the [documentation](https://reactnative.dev/docs/flatlist)